.eslintrc.js

```diff
@@ -1,85 +1,23 @@
-module.exports = {
+'use strict';
+
+const config = {
   root: true,
-  parserOptions: {
-    ecmaVersion: 2019,
-    sourceType: 'module',
-    ecmaFeatures: {
-      jsx: true
+  env: { commonjs: true, node: true, browser: true },
+  extends: ['ackama', 'ackama/react'],
```

Comment:
Is it safe for us to extend `ackama/react` if we're not necessarily setting up webpack?

Reply:
I think so, we were already extending the default set of react rules so I dont think using our specific set of rules should cause anything to start error-ing.

Reply:
If by "safe" you mean "will it error": No - the rules don't use anything from react packages themselves, so can lint projects that try to use react even if they don't support it (which will error outright separately).

It could be slightly confusing, but I don't think it's anymore so than us using `rubocop-rails` in our rubocop config, meaning apps need that even if they're not using rails.